### DISCUSSION TITLE
Reproductions

### DISCUSSION THREAD

**Discussion by lake44me** [crystalyragui/MARC2RDA#385] [General]
**Reproductions (reprints, photocopies, etc.) - discussion page in progress :-).**

- Question 1.  What data in a MARC21 record reliably indicates that what the record describes is a reproduction of another resource? 
- Question 2.  If the MARC21 record indicates that what is being described by the record as a whole is a reproduction, how should the mapping/conversion be different from a non-reproduction? 
- Question 3.  What data should be provided describing the expression, manifestation, or (rda) item being reproduced? 
- Question 4   What RDA property should be used to relate a reproduction (manifestation, item?) to an entity that it reproduces?  

***Question 1. MARC21 fields that refer to one or more kinds of reproductions*** 

- Tag 533 Reproduction Note.  Definition:  "Descriptive data for a reproduction of an original item when the main portion of the bibliographic record describes the original item and the data differ".

- Tag 008 position 6 (Type of Date/Publication Status) for all formats

code "r" - Reprint/reissue date and original date
008/07-10 contain the date of reproduction or reissue; 008/11-14 contain the date of the original, if known. 008/11-14 contain code u ("uuuu"), if unknown.

If multiple dates are available for the original publication, 008/11-14 contain the earlier date. With original photographic material, the work being described would be a later photoprint made from an earlier created photonegative. With original prints, the work being described would be a restrike made from the (usually deceased) artist's earlier-created plate or block.

Notes at beginning:

"For continuing resources, 008/07-10 contain the beginning date of publication (chronological designation) and 008/11-14 contain the ending date. For reprints of serials and for reproductions of serials being described in the body of the entry, the beginning and ending dates of the original are input in these character positions."

>008/06	r
008/07-10	1983
008/11-14	1857
260	##$aBoston :$b[s.n.,$c1983?]
500	##$aOriginal version: Pittsburg, Pa. : W'm Schuchman & Bro. Lith., [1857].<

>008/06	r
008/07-10	1966
008/11-14	uuuu
500	##$aReprinted from Green Howard's Gazette.
[Original date of publication is unknown.]<

>008/06	r
008/07-10	uuuu
008/11-14	1963
260	##$aNew York :$b[s.n.,$cn.d.]
500	##$aPreviously published in 1963.<

- MARC21 008 position 23 Form of Item for BOOKS:

r - Regular print reproduction
Eye-readable print, such as a photocopy.

008/23	r
500	##$aPhoto-reproduction of 1963 edition.

r - Regular print reproduction
Eye-readable print, such as a photocopy.

r - Regular print reproduction
Eye-readable print, such as a photocopy.

008/23	r
500	##$aPhotocopy of the original score composed in 1877.

- MARC21 008 position 23 Form of Item for CONTINUING RESOURCES:

r - Regular print reproduction
Eye-readable print, such as a photocopy.

- Tag 843 Reproduction Note (defined in MARC21 Format for Holdings):  "Descriptive data for a reproduction of an original item when the main portion of the bibliographic record describes the original item and the data differ."

Although the MARC21 Format uses the word "item" to describe what is reproduced, we should not assume this refers to the RDA /LRM definition of an Item entity. 

- Tag 534 Original version note.  Definition:  Descriptive data for an original item when the main portion of the bibliographic record describes a reproduction of that item and the data differ. Details relevant to the original are given in field 534.

The resource being cataloged may either be a reproduction (e.g., scanned image, or PDF), or an edition that is similar enough that it could serve as a surrogate for the original (e.g., HTML).

- Tag 775  Other edition entry.  Definition:  Entry for another available edition of the target item (horizontal relationship).  
This tag may be used to describe a reproduction relationship using $i Relationship information (textual) and/or $4 Relationship (URI for a relationship, e.g. RDA Registry relationship such as http://rdaregistry.info/Elements/m/P30043 Reproduction of item of

- 007 byte 5 for Map format,  code f  "Facsimile".

**Comment by CECSpecialistI** [comment]
Reprints are not reproductions; date code "r" should not be included here.

**Reply by lake44me** [reply]
I think I understand about reprints and reissues (at least by the same publisher).  But it appears code "r" also gets used for reproductions that aren't reprints or reissues.  See Yale's cataloging manual section on Reproductions (from 2016, following RDA of the time):  https://web.library.yale.edu/cataloging/reproductions-rda/facsimiles-reprints.  It instructs to use 008 position 6 code "r" for reproductions and gives an example. 
And, perhaps it's in scope for this discussion to include proper RDA modeling for reprints/reissues - 'cause as I'm looking at the elements/property definitions, I'm getting confused.

**Reply by CECSpecialistI** [reply]
@AdamSchiff is Yale's practice here correct? I haven't cataloged enough reproductions to really know, but I thought 008/6 code "r" was only for reprints/reissues and not for reproductions.

**Comment by CECSpecialistI** [comment]
Practice is variable based on time of record creation and cataloging community.

**Reply by lake44me** [reply]
I added tags 534 and 775 to the list of tags that may indicate a reproduction relationship.

**Comment by AdamSchiff** [comment]
If you compare the MARC documentation with OCLC Bibliographic Formats and Standards, you'll see that OCLC more clearly limits code r to reprints/reissues. The MARC format itself uses the word reproduction in the definition, which OCLC doesn't repeat.

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by CECSpecialistI** [comment]
007 $f in Maps format

**Reply by lake44me** [reply]
Added, fwiw.

**Comment by lake44me** [comment]
@AdamSchiff I think Yale's use of the 008 byte 6 date type "r" is practical (to be able to put both original and reproduction dates in the 008.  Since the 008 dates are usually the first date source for discovery system indexing, this enables surfacing the original date for indexing and display so users looking for the original using date in the search qualifiers (or looking at dates for various versions) can find and identify the "expression" even though reproduction information is also displayed.   This might not square with OCLC instructions, however.

OCLC's instructions do seem to differ quite a bit from LC's, both in their instructions for use of 008 byte 6 "r" , specification of use of 533 and also their introduction of tag 539 Fixed-Length Data Elements of Reproduction Note (R) which isn't in MARC21.  @CECSpecialist I I'm adding tag 539 to the list of MARC fields we may encounter, and I suppose we need a mapping sheet for it since we could expect to encounter it in our conversion of records originating from OCLC.

OCLC's date type "r" [instructions](https://www.oclc.org/bibformats/en/fixedfield/dtst.html) are lengthy and link out to other documents so I won't reproduce them here.  I'll attempt to incorporate important differences into the document above.  OCLC'S "provider neutral" policy for electronic and print-on-demand manifestations of same content, which I have not worked with since it was extended from serials to include monographs, seems to be a factor here.  (gnashes teeth).   

OCLC uses the word "republication" without definition.  It's not clear to me if it's a condition under which date type "r" would be applied (with the original date in "Date 2" and the republication date in "Date 1").    "If a resource qualifies as a republication and only slight changes have been made (e.g., minor corrections or new material has been added, such as an introduction, preface, foreword), treat the resource as a republication."  Does anyone have background on this - or should we consult other catalogers?  

I would think new material would qualify it as a new expression, but perhaps not "minor corrections"...

It's not spelled out, but publication in a different form from the original seems to NOT be a condition for using date type "r", at least in certain cases.  
Exclusions for use of "r" include:
- Microform reproductions (including theses and dissertations whether previously published or not). Code DtSt and Dates for the original. Enter date information for the microform reproduction in [field 533](https://www.oclc.org/bibformats/en/5xx/533.html) and in [field 539](https://www.oclc.org/bibformats/en/5xx/539.html) subfields ǂa, ǂb, and ǂc.
- Electronic reproductions follow provider-neutral guidelines. For more information, see chapter 3.1, [Provider-Neutral Cataloging, Online resources](https://www.oclc.org/bibformats/en/specialcataloging.html#onlineresources).
- Print reproductions follow provider-neutral guidelines. For more information, see chapter 3.1, [Provider-Neutral Cataloging, Print-on-demand publications and photocopies](https://www.oclc.org/bibformats/en/specialcataloging.html#printondemandpublicationsandphotocopies).
- For motion pictures, change in form (e.g., a motion picture re-released as a videorecording)
 
All of this is leading me to think that, while it is a clue, we cannot rely on Date type "r" in the 008 to signify that the manifestation "described by the record as a whole" is a reproduction of another manifestation, nor to indicate which data elements describe the manifestation "of the record" and which elements describe a manifestation that was reproduced.  Presence of particular tags, particularly 533, 534 or 775, should be the mapping condition.  The next area of discussion is how and to what extent do we populate Expression properties from these records and whether it will be necessary to mint another Manifestation identifier, supply properties for it from the record, and specify a relation between the reproduction and the reproduced, in order to capture what's in the record.

Date type "r" is mainly useful to identify what entity's date is in the 008 bytes 11-14 (Date 2), should we choose to map it.  Right now, it looks like we are not mapping Date 2 when the code is "r".  
  
@SitaKB @GordonDunsire @gerontakos @pan-zhuo  @JianPLee @junghaelee @briesenberg07

**Comment by lake44me** [comment]
OCLC documentation makes reference to LC-PCC PS for 1.11 Facsimiles and Reproductions  https://original.rdatoolkit.org/lcpschp1_lcps1-99044.html .

**Comment by CECSpecialistI** [comment]
Please upload examples of reproduction records here for @lake44me to compile and use for potential mapping examples of reproductions in RDA

**Comment by CECSpecialistI** [comment]
Sita mentioned 535 and 776 during meeting today

**Comment by szapoun** [comment]
From the meeting notes [1st March](https://github.com/uwlib-cams/MARC2RDA/wiki/2023-Meeting-Minutes#march-1-2023-800am---930am-pst)

008 byte 6 date type "r" does not always imply reproduction. It depends on the cataloguing policy. 

533 tag describes the reproduction, but there is not a consistency regarding other tags. 245, 264, 3XX may describe the original manifestation or the reproduced one.

CY. Can we automatically detect which tags refer to the original and which to the reproduction? If 775 exists, there is certainty about what the records describes, i.e., the original manifestation or the reproduced one.

SZ. Proposed to use 534 just as a note, and not to mint new manifestations. 

GD. The answer to CY's question is no, since there has been no regular practice in the past.

LA. Since in US most libraries use PCC or OCLC guidelines, it might be helpful to make a guess using real examples. 

AS. A prossible problem would be with provider-neutral records (040$epn) used in Print On Demand cases.

LA. Will try to mock something up and asked for examples (reprints, facsimiles, reproductions) by the members of the WG.  

SB. Mentioned 535 & 776 used by the National Library of Netherlands

**Comment by lake44me** [comment]
## Elements in the RDA toolkit related to reproductions, reprints or facsimiles

### Manifestation -> Manifestation relations

All options for recording manifestations are the same:  Record as value of appellation of manifestation, as unstructured description as value of title of manifestation, a structured description as value of access point of manifestation, an identifier as a value of Identifier of manifestatiom, or an IRI for a related manifestation as a RWO  

- **electronic reproduction of manifestation of** http://rdaregistry.info/Elements/m/P30136 "is electronic reproduction of manifestation of" A manifestation in an analog format that is transferred to a digital format. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-64cfdd41-8e04-32c3-b7c2-bba805839dd3

- **electronic reproduction of manifestation** http://rdaregistry.info/Elements/m/P30016 "has electronic reproduction of manifestation"   A manifestation in a digital format that is the result of the transfer of a manifestation in an analog format.   
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-e85714dd-247d-39d7-8631-5322ac924afd

- **facsimile of manifestation** http://rdaregistry.info/Elements/m/P30031 "has facsimile of manifestation" A manifestation that exactly reproduces another manifestation
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-5f41752f-44f5-30d8-b17c-9a424a3d5637

- **facsimile of manifestation of** http://rdaregistry.info/Elements/m/P30047 "is facsimile of manifestation of" A manifestation that is used as the basis for an exact reproduction.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-a8671732-4aed-3b2c-9b6e-310e4617a9cf

- **facsimile part** http://rdaregistry.info/Elements/m/P30046 "has facsimile part" A manifestation that is exactly reproduced as a part of another manifestation.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-715f5d09-8d7d-3542-be2d-9ca06c61f019

- **facsimile part of** http://rdaregistry.info/Elements/m/P30019 "is facsimile part of" A manifestation that is a part manifestation that exactly reproduces another manifestation. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-c699e54f-0b4c-3a92-9448-ff1263286306

- **reprint of manifestation of** http://rdaregistry.info/Elements/m/P30093 "is reprint of manifestation of" A manifestation that is a printed manifestation that is used as the basis for a reissue of a manifestation.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-d83588be-4edb-309c-8ace-ac9bc6311d2b 

- **reprinted as manifestation** http://rdaregistry.info/Elements/m/P30038 "is reprinted as manifestation" A manifestation that is a reissue of another printed manifestation. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-0c96bf50-cb2c-32ed-a67e-f05576ab0b3e

- **reproduced as manifestation** http://rdaregistry.info/Elements/m/P30039 ""is reproduced as manifestation" A manifestation that reproduces another manifestation. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-cd3d801c-f0d6-32a5-ae54-de169aa61a5e

- **reproduction of manifestation of** http://rdaregistry.info/Elements/m/P30043 "is reproduction of manifestation of" A manifestation that is used as the basis for a reproduction.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-f2183d7c-7677-3b6d-835f-615839185e15

### Manifestiation -> Item relations

All options for recording items are the same:  Record as value of appellation of item, as unstructured description as value of title of item, a structured description as value of access point of item, an identifier as avalue of Identifier of item, or an IRI for a related item as a RWO

- **electronic reproduction of item of** http://rdaregistry.info/Elements/m/P30299 "is electronic reproduction of item of" Relates a manifestation to an item in an analog format that is transferred to a manifestation in a digital format.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-8b3f2d81-e0b3-35a6-9cb2-bb0b0009bac4 

-  **facsimile of item of** http://rdaregistry.info/Elements/m/P30300 "is facsimile of item of"  An item that is used as the basis for an exact reproduction.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-bab90bcb-bb61-3147-bc20-4d578ba75639   

- **reprint of item of**  http://rdaregistry.info/Elements/m/P30302  "is reprint of item of" An item that exemplifies a printed manifestation that is used as the basis for a reissue of a manifestation. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-af5e470c-9834-32cc-9b39-2b334dd87bb7

- r**eproduction of item of** http://rdaregistry.info/Elements/m/P30303 "is reproduction of item of" An item that is used as the basis for a reproduction.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-ca7c8111-5f9e-323d-b0bf-a167ca0259c4


### Item -> Manifestation relations

All options for recording manifestations are the same:  Record as value of appellation of manifestation, as unstructured description as value of title of manifestation, a structured description as value of access point of manifestation, an identifier as a value of Identifier of manifestatiom, or an IRI for a related manifestation as a RWO 

- **digital transfer of item** http://rdaregistry.info/Elements/i/P40087 "has digital transfer of item" A manifestation in a digital format that results from the transfer of an item in another digital format.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-966adbee-be6b-38ab-b012-d5ef855829ca

- **electronic reproduction of item** http://rdaregistry.info/Elements/i/P40088 "has electronic reproduction of item" A manifestation in a digital format that is the result of the transfer of an item in analog format. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-df0cb73e-2752-3ae3-b849-03764fbdbc1e

- **facsimile of item** http://rdaregistry.info/Elements/i/P40089  "has facsimile of item" A manifestation that exactly reproduces an item that exemplifies another manifestation.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-df0a6c8e-ed76-3682-8ee0-81fd4827f02b

- **preservation facsimile of item** http://rdaregistry.info/Elements/i/P40090 "has preservation facsimile of item" A manifestation that consists of an exact reproduction of an item on preservation-quality media, such as acid-free permanent or archival paper.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-6eb23766-5f10-3c32-9907-cbda194bb9e8

- **reprinted as item** http://rdaregistry.info/Elements/i/P40091 ""is reprinted as item" A manifestation that is a reissue of an item that exemplifies another printed manifestation.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-f7922cad-3080-349f-9e75-52da85e7cb4d

- **reproduced as item** http://rdaregistry.info/Elements/i/P40092 "is reproduced as item" A manifestation that reproduces an item that exemplifies another manifestation. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-55308b7f-24f1-386e-886a-5cdc21cedc02


### Item -> Item relations
I couldn't find any that relate to reproduction

### Other relations I'm curious about:

- **equivalent item** http://rdaregistry.info/Elements/i/P40031 "has equivalent item" An item that exemplifies a manifestation that embodies the same expression of a work. 
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-2d57c85f-20f3-3335-9d0f-1672adb8ff9d
Under what circumstances would this be useful?  If you didn't want to do a full WEMI stack for the other manifestation but just wanted to relate the item to a manifestation you had already described without getting into the details?

**Reply by lake44me** [reply]
The following is lengthy but attempts to lay out mapping decisions and conditions for records when a 533 tag is present.  It mentions but doesn't address, mapping for other paradigms for describing reproductions in MARC.  It's not complete but I wanted to get the general ideas in front of you.  My questions, does this sound feasible, how would we implement it, and are there things I've missed?

I have captured a few records with 533 tags (and some reproductions without 533 tags) from our catalog, and will attach them later as readable PDFs - can also supply more and make MARCXML available if that would be helpful.  I think others were supposed to post some examples as well - can I call on you?  Particularly if you can find ones with 533 that don't fit this "patterns" as described in OCLC  Bib Formats and Standards Special Cataloging section descriptions for microforms and electronic resources.  

## MARC21 to RDA for records containing a 533 tag

### Condition:  If record contains a 533 tag
-   Map description of the **manifestation described by the record** (repromanif) with respect to any subfields present in the 533.  (See below)   
-   Also create a manifestation IRI for the "original" (originalmanif) which is reproduced by the repromanif.  
-   Based on presence of 533, some tags in the record may be assumed to describe both repromanif and originalmanif, so they can be mapped accordingly.  (See below)
-   Some tags in the record can be assumed to describe originalmanif only and are mapped accordingly (see below)
-   Other tags in the record are defined to describe the reproduction and shcould be mapped to repromanif (see below) 

### Reproduction to original manifestation relations	  
-   Relate repromanif to originalmanif using the appropriate property.  
-   Condition:   If there is an indication in 533 subfield a or elsewhere in the record that the reproduction is "electronic", use RDA property  **is electronic reproduction of manifestation of"" http://rdaregistry.info/Elements/m/P30136 to relate repromanif to originalmanif
--	evidence of this condition, presence of string "electronic" in text of 533 $a	
--  if not present, other evidence of this condition:  ?????
-	Condition:  If there is an indication in 533 subfield a or elsewhere in the record that the reproduction is a "facsimile", use RDA property **is facsimile of manifestation of** http://rdaregistry.info/Elements/m/P30047 to relate repromanif to originalmanif  (note:  this may not exist - likely those would not have 533 tags - have not found example) 

-   Otherwise, use property   "is reproduction of manifestation of" http://rdaregistry.info/Elements/m/P30043 

I am assuming that from the MARC we would not be able to determine whether a record with a 533 is a reproducction of a specific copy and use those relations.  If there is also a 535 field with first indicator 1, identifying a source repository for the original(s) of the manifestation being described, we might consider how that might relate to the originalmanif record (later...)  
	
	
### Tags that can be assumed to map to both repromanif and originalmanif, if desired.  

We could have a discussion about how much of the data to map to originalmanif.  I think we have the choice to not even mint an IRI and simply give the title as an unstructured string, but that would not serve users well.  The originalmanif mainly serves to relate the original to the reproduction, to enable users to identify what the reproduction is reproducing.  It's possible that a given collection might also have already a manifestation record for that original, but adjusting the linkage to use that manifestation's IRI would have to be a post-conversion task.  I think we can and should be selective, and of course, with many of the note fields, we probably couldn't tell whether they will apply to the original or the reproduction or both.    

-   100, 110, 111 (creative agent)  
-   245  Title  
-   336  Content type - if it's truly a reproduction, this should by definition be the same    


### Tags that could be overriden by 533 subfields should be mapped to originalmanif only.

- 260 or 264 all subfields, to originalmanif.  (Note:  if the 533 is lacking subfields b (place of reproduction), c (Agency responsible for reproduction), d (date of reproduction), that is information we don't have for the description of repromanif.  Presence of 533 should indicate that the record is following the practice of describing the original.) 
- 300 all subfields, to originalmanif.  (Note:  if the 533 is lacking subfield e (Physical description), that is information we don't have about repromanif .
- 490 or 830 (series fields) to originalmanif.  (Note:  the 533 may or may not have $f Series statement of reproduction - this would apply to repromanif, but can't say that manifestation was published in the original series unless a 533 $f says so.)

### Other MARC elements that can be assumed to describe the originalmanif only, see LC-PCC PS for 1.11 in Original RDA toolkit  https://original.rdatoolkit.org/lcpschp1_lcps1-99044.html. 

Note that the PCC policy relates to microforms, photocopies and print on demand manifestations, not electronic reproductions, but the PCC Provider-Neutral E-Resource MARC Record Guidelines, the latest version of which can be found as a Word doc here: https://www.loc.gov/aba/pcc/scs/documents/PCC-PN-guidelines.html, and which addresses e-resources that reproduce print or other formats, only permits use of the 533 tag under certain circumstances.  "Use only for records for DLF Registry of Digital Masters, HathiTrust Digital Library, and other digital preservation projects. Use with $5".  However, many MARC records supplied in batch by vendors or other sources for electronically reproduced physical resources follow the "describe the original, use 533" paradigm (whether coded RDA, AACR2 or something else).  Outside the scope of this mapping logic draft for 533 is mapping of "Provider neutral" records, which should be identifiable by containing an 040 $e pn Note that these records may be for resources only published electronically, possibly provided by multiple agencies (aggregators, etc.), as well as resources reproducing materials previously published in another form, and resources simultaneously published in electronic and another form.  

- 008 07-10 and 11-14 describe dates for the original (per LC-PCC PS 1.11  - 008 byte 07 is not to be coded "r".  If it is, we could consider that as not fitting the pattern we are mapping for 533 - not sure whether we can map those ??????) 

- other 008 positions and fixed fields - need to examine whether mappings need to be for originalmanif or repromanif.  OCLC guidelines https://www.oclc.org/bibformats/en/specialcataloging.htm for microforms and electronic resources may be best rule of thumb.  

- 01x - 09x - I need to review these in light of how we are mapping them (at expression or manifestation level?) but if at manifestation level, probably would describe the original (particularly if the repromanif record was cloned from, say, a print record for the original).  

- 222 Key title

- 250 edition statement

- 255 Cartographic mathematical information

- 776 (this is optional in LC-PCC PS for 1.11).  (mapping to come, but note to use same originalmanif URI created based on presence of 533 for any additions to description from this field)


### Tags that should be mapped to repromanif only

- 020 ISBN.  OCLC guidelines for microforms say to enter an ISBN for the microform reproduction in 020, but put any ISBN assigned to the original resoure in 776 $z.

- 337 Media type - PCC PS says to code for the reproduction media type

- 338 Carrier type - PCC PS says to code for the reproduction carrier


### Mapping of subfields in 533 to repromanif:

-  $3 Materials specified - indicates that original is an aggregate?  of which this reproduction is of a part.  Use note on manifestation http://rdaregistry.info/Elements/m/datatype/P30137, provide this text prefix as context:  "Reproduces this part of the original: (content of $3) 

-  $a Type of reproduction.  Map to note on manifestation http://rdaregistry.info/Elements/m/datatype/P30137 with prefix "Type of reproduction:  (content of $a)

-  $b Place of reproduction.  Map to place of publication (unstructured)  http://rdaregistry.info/Elements/m/datatype/P30088

-  $c Agency responsible for reproduction  Map to Publisher agent (unstructured) http://rdaregistry.info/Elements/m/datatype/P30083 

-  $d Date of reproduction.  Map to has date of publication  http://rdaregistry.info/Elements/m/datatype/P30011  (other option would be date of production but we can't be sure and publication is more likely) 

-  $e Physical description of reproduction.  Map to ?????? - problematic because this is entire 300 tag, not just $a.  Note on manifestation, or we could try to parse out the subfield content if it has ISBD punctuation, and map as if it was in a 300 tag.

-  $f Series statement of reproduction  Map to has series statement http://rdaregistry.info/Elements/m/datatype/P30106

- $m Dates and/or sequential designation of issues reproduced - ??????????  probably extent of manifestation http://rdaregistry.info/Elements/m/P30182 according to instructions in RDA toolkit if I read them right. If a microform or electronic manifestation "... is in a format that parallels a print, manuscript, or graphic counterpart [...] specify the number of subunits by applying the instructions for the appropriate parallel counterpart" .  But, if so, where would we put 533 $e physical description of reproduction, which gives the unit and number of, e.g., microfilm reels?   This is more an issue for serials which we aren't mapping yet, but it could apply to multipart monographs as well.  
   
- $n Note about reproduction. Map to note on manifestation http://rdaregistry.info/Elements/m/P30137  

... to be continued...  Subfields $y (data provenance), $5 (Institution to which field applies), $6 (Linkage) and $7  Fixed-length data elements of reproduction present special challenges for this tag's mapping.   $8 (Field link and sequence number) can be handled as specified in decisions.

**Reply by GordonDunsire** [reply]
Many of the RDA "reproduction" elements are pre-3R legacy with fuzzy semantics that are intended to be sufficient to distinguish one "kind" of reproduction from another.

The base term "reproduction" is defined as "An exact copy of the content of a manifestation or item made by mechanical or electronic means". What "exact copy of the content" means is moot; it is intended to cover microfilm reproductions, digital reproductions, etc.

The definition is made in the context of aggregates. A manifestation that reproduces a monograph and adds a new introduction is an aggregate, not a reproduction.

This why the legacy "part reproduction" elements were removed during 3R; the "part facsimile" elements apply to manifestation sub-units (aka parts).

LRM does not recognize "part item"; RDA treats "contained in item" as a shortcut.

"Bibliographic history" is a combination of manifestation reproduction and expression re-combination :-)

**Comment by lake44me** [comment]
Examples of records with 533:
[Record View 9937587698202486 Emory Unbekannte altarwerke von Durer und grunewald in nordingen und wimpfen print only subf a.pdf](https://github.com/uwlib-cams/MARC2RDA/files/10994008/Record.View.9937587698202486.Emory.Unbekannte.altarwerke.von.Durer.und.grunewald.in.nordingen.und.wimpfen.print.only.subf.a.pdf)
[Record View 990004615080302486 Emory Extracts from Boldon Book, or the Domesday of the county of Durham ECCO.pdf](https://github.com/uwlib-cams/MARC2RDA/files/10994009/Record.View.990004615080302486.Emory.Extracts.from.Boldon.Book.or.the.Domesday.of.the.county.of.Durham.ECCO.pdf)
[Record View 990007369550302486 Emory A year in Canada and other poems Alexander Street series.pdf](https://github.com/uwlib-cams/MARC2RDA/files/10994010/Record.View.990007369550302486.Emory.A.year.in.Canada.and.other.poems.Alexander.Street.series.pdf)

**Comment by lake44me** [comment]
@CECSpecialistI et. al. here is a link to a summary of the recommendations for handling "conditions for reproductions" for records using the PCC approach to multi-manifestation description using 533 and 588 conditions to identify them.  These came out of the subgroup meeting August 15 and my meeting with Doreen on the August 19.

https://docs.google.com/document/d/1cly9kfJViZOEwc7hoYIhUM2zVkdVudSU_LUWxodcOAw/edit?usp=sharing
